<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkloadWindow</class>
 <widget class="QDialog" name="WorkloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tb_workloadType">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="t_workloadPerYear">
      <attribute name="title">
       <string>Загруженность за год</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="cp_graphicPerYear" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="t_workloadPerMonth">
      <attribute name="title">
       <string>Загруженность за месяц</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QCustomPlot" name="cp_graphicPerMonth" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_months"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lb_airportName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>WorkloadWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
